### PR TITLE
HOT-FIX: Fix broken links

### PR DESCRIPTION
**Pull request by bbejeck**
Found several broken links in the docs, mostly in `streams-concepts`

For testing, I ran kafka-site locally with these changes.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by bbejeck** [comment]
@guozhangwang, @mjsax, @vvcephei for review

\cc @joel-hamill @JimGalasyn  for reviews as well

**Review by joel-hamill — APPROVED** [review]
I tested this locally - changes look good. Is there any new content added here? Hard to tell from the diff. If so, just wanted to make sure it is also added to CP.

**Comment by bbejeck** [comment]
>Is there any new content added here?

@joel-hamill yes, the sections added are `KStream`, `KTable`, and `GlobalKTable` as the links were there but not the content.  But this info already exists in CP.

**Comment by joel-hamill** [comment]
@bbejeck lgtm...thanks

**Comment by bbejeck** [comment]
Fixed some additional links, should be all of them now.

File: docs/streams/core-concepts.html [hunk 1]
```diff
@@ -131,6 +235,54 @@ <h3><a id="streams_time" href="#streams_time">Time</a></h3>
...
+    <h3><a id="streams_concepts_aggregations" href="#streams_concepts_aggregations">Aggregations</a></h3>
+    <p>
+        An <strong>aggregation</strong> operation takes one input stream or table, and yields a new table by combining multiple input records into a single output record. Examples of aggregations are computing counts or sum.
+    </p>
+
+    <p>
+        In the <code>Kafka Streams DSL</code>, an input stream of an <code>aggregation</code> can be a KStream or a KTable, but the output stream will always be a KTable. This allows Kafka Streams to update an aggregate value upon the late arrival of further records after the value was produced and emitted. When such late arrival happens, the aggregating KStream or KTable emits a new aggregate value. Because the output is a KTable, the new value is considered to overwrite the old value with the same key in subsequent processing steps.
+    </p>
+
+    <h3 <a id="streams_concepts_windowing" href="streams_concepts_windowing">Windowing</a></h3>
```
Comment (guozhangwang) [inline docs/streams/core-concepts.html]: typo? `<h3>

File: docs/streams/core-concepts.html [hunk 2]
```diff
@@ -88,6 +88,110 @@ <h3><a id="streams_topology" href="#streams_topology">Stream Processing Topology
         At runtime, the logical topology is instantiated and replicated inside the application for parallel processing (see <a href="/{{version}}/documentation/streams/architecture#streams_architecture_tasks"><b>Stream Partitions and Tasks</b></a> for details).
     </p>
 
+    <h3><a id="streams_concepts_kstream" href="#streams_concepts_kstream">KStream</a></h3>
```
Comment (guozhangwang) [inline docs/streams/core-concepts.html]: I suggest moving this part to the developer_guide / DSL subsection, as they are particularly for the DSL itself not for general Streams concepts.

I know in CP docs they are indeed listed under https://docs.confluent.io/current/streams/concepts.html but personally I'm not in favor of doing that for the above reason.

File: docs/streams/core-concepts.html (hunk 1 above)
Comment (bbejeck) [reply docs/streams/core-concepts.html]: ack

File: docs/streams/core-concepts.html (hunk 2 above)
Comment (bbejeck) [reply docs/streams/core-concepts.html]: ack

**Comment by bbejeck** [comment]
@guozhangwang made changes per comments.

@joel-hamill mind taking another look? I've tested locally, but another quick pass wouldn't hurt.

**Comment by guozhangwang** [comment]
LGTM. waiting for @joel-hamill for a final pass.

File: docs/streams/core-concepts.html [hunk 3]
```diff
@@ -244,7 +140,7 @@ <h3><a id="streams_concepts_aggregations" href="#streams_concepts_aggregations">
         In the <code>Kafka Streams DSL</code>, an input stream of an <code>aggregation</code> can be a KStream or a KTable, but the output stream will always be a KTable. This allows Kafka Streams to update an aggregate value upon the late arrival of further records after the value was produced and emitted. When such late arrival happens, the aggregating KStream or KTable emits a new aggregate value. Because the output is a KTable, the new value is considered to overwrite the old value with the same key in subsequent processing steps.
     </p>
 
-    <h3 <a id="streams_concepts_windowing" href="streams_concepts_windowing">Windowing</a></h3>
+    <h3> <a id="streams_concepts_windowing" href="streams_concepts_windowing">Windowing</a></h3>
```
Comment (joel-hamill) [inline docs/streams/core-concepts.html]: this should be (missing `#`)

```
<h3> <a id="streams_concepts_windowing" href="#streams_concepts_windowing">Windowing</a></h3>
```

**Review by joel-hamill — APPROVED** [review]
One anchor link to fix

**Comment by bbejeck** [comment]
updated this, fixed anchor link

**Comment by guozhangwang** [comment]
Thank you @bbejeck !!

**Pull request merged**